### ISSUE TITLE
bug in getInvalidDatasetMembers.R

### ISSUE THREAD

**Issue by vshioshvili** [jason-p-pickering/datim-validation#46]
When trying to supply data sets via parameter, it generates an error

```
 Error in match(x, table, nomatch = 0L) : object 'allDataSets' not found 
```

It is caused by 
https://github.com/jason-p-pickering/datim-validation/blob/master/R/getInvalidDatasetMembers.R#L12

allDataSets is only created when parameter is not provided, and then code attempts to use it.

**Comment by jason-p-pickering** [comment]
@vshioshvili yes, you are right. I have completely rewritten these methods, but its a merge I have not managed to make yet. Will try and fix this however for this version

**Comment by jason-p-pickering** [comment]
@vshioshvili I think this has been fixed in 2428de4d3ff124e. Could you give it a try and let me know? Still working on unit tests there.
